__tests__/seed.test.js

```diff
@@ -75,6 +75,29 @@ describe('GET /api/articles', () => {
     });
 });
 
+describe('GET /api/articles/?topic', () => {
+    test('200: filters the articles by the topic value specified in the query.', () => {
```

Comment:
One more 200 test needed - what happens if we get a topic which exists but has no articles?